### PR TITLE
fix: add missing LOG_LEVEL for the project

### PR DESCRIPTION
**Pull request by polaroi8d**

### CODE REVIEW

**Review by robot9706 — APPROVED** [review]

File: .env.example [hunk 1]
```diff
@@ -22,7 +25,7 @@ KRATOS_SECRET=Random_Generated_String
 # Mailserver settings
 SMTP_URI=smtps://username:password@mailserver.example.com:465
 FROM_EMAIL=from@example.com
-FROM_NAME=dyrectorio Platform
+FROM_NAME="dyrectorio Platform"
```
Comment (m8vago) [inline .env.example:28]: is it necessary?

File: docker-compose.yaml [hunk 2]
```diff
@@ -25,6 +25,7 @@ services:
     command: [serve]
     environment:
       - TZ=${TIMEZONE:-UTC}
+      - LOG_LEVEL=${LOG_LEVEL:-debug}
```
Comment (m8vago) [inline docker-compose.yaml:28]: we have a docker-compose.http-only.yaml as well

**Review by m8vago — CHANGES_REQUESTED** [review]

File: .env.example (hunk 1 above)
Comment (polaroi8d) [reply .env.example:28]: Auto format. I'll remove it.

File: docker-compose.yaml (hunk 2 above)
Comment (polaroi8d) [reply docker-compose.yaml:28]: Oh, yeah, is anyone using that? 🦅

Comment (m8vago) [reply docker-compose.yaml:28]: yes

**Review by m8vago — APPROVED** [review]

**Pull request merged**